src/Concerns/PerformsCharges.php

```diff
@@ -57,4 +58,49 @@ public function refund($paymentIntent, array $options = [])
             $this->stripeOptions()
         );
     }
+
+    /**
+     * Begin a new Checkout Session.
+     *
+     * @param  int  $amount
+     * @param  string  $name
+     * @param  int  $quantity
+     * @param  array  $sessionOptions
+     * @param  array  $customerOptions
+     * @return \Laravel\Cashier\Checkout
+     */
+    public function checkout($amount, $name, $quantity = 1, array $sessionOptions = [], array $customerOptions = [])
+    {
+        return Checkout::create($this, array_merge([
+            'line_items' => [[
+                'price_data' => [
+                    'currency' => $this->preferredCurrency(),
+                    'product_data' => [
+                        'name' => $name,
+                    ],
+                    'unit_amount' => $amount,
+                ],
+                'quantity' => $quantity,
+            ]],
+        ], $sessionOptions), $customerOptions);
+    }
+
+    /**
+     * Begin a new Checkout Session.
+     *
+     * @param  string  $price
+     * @param  int  $quantity
+     * @param  array  $sessionOptions
+     * @param  array  $customerOptions
+     * @return \Laravel\Cashier\Checkout
+     */
+    public function checkoutProduct($price, $quantity = 1, array $sessionOptions = [], array $customerOptions = [])
+    {
+        return Checkout::create($this, array_merge([
+            'line_items' => [[
```

Comment:
why the double array [[ ?

Reply:
line items is an "array of hashes" or in this case an array of associative arrays. So the outer `[` is for the "array of" and the second `[` is for the first "hash" in the list.

Reply:
got it thanks!